chore: Federation editorials and acknowledgments (#129)
italia/eidas-it-wallet-docs: docs/en/trust.rst

     - **Description**
     - **Required**
   * - **authority_hints**
     - Array of URLs (String). It contains a list of URLs of the immediate superior entities, such as the Trust Anchor or

>        an Intermediate, that issue an Entity Statement related to this subject.

Reviewer: ```suggestion
       an Intermediate, that issues an Entity Statement related to this subject.
```

Entity Statements
-----------------

> Trust Anchors and Intermediates publish an Entity Statement related to an immediate Subordinate.

Reviewer: ```suggestion
Trust Anchors and Intermediates publish an Entity Statement related to a direct Subordinate.
```

Author: usiamo immediate in oidc fed, uso lo stesso termine

Wallet Instance Attestation
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The Wallet Provider issues the Wallet Instance Attestation, certifying the operational status of its Wallet Instances and including one of their public keys. 

The Wallet Instance Attestation contains the Trust Chain that attests the validity for its issuer (Wallet Provider).

> The Wallet Instance presents its Wallet Instance Attestation within the signed request during the PID issuance phase, containing the Trust Chain related to the Wallet Provider. The PID Provider issues a PID including the public key contained in the Wallet Instance Attestation, this produces the Holder Key Binding within the issued PID.

Reviewer: The Holder Key Binding is actually not done with the public key contained in the WIA.

Author: yes, deprecation here

Author: done

> The Wallet Instance presents its Wallet Instance Attestation within the signed request during the PID issuance phase, containing the Trust Chain related to the Wallet Provider. The PID Provider issues a PID including the public key contained in the Wallet Instance Attestation, this produces the Holder Key Binding within the issued PID.

Author: ```suggestion
The Wallet Instance presents its Wallet Instance Attestation within the signed request during the PID issuance phase, containing the Trust Chain related to the Wallet Provider. The PID Provider issues a PID including the public key provided by the Wallet Instance to have the Holder Key Binding within the issued PID.
```

Offline EUDI Wallet Trust Attestation
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Given that a mobile device SHOULD not publish its metadata online at the *.well-known/openid-federation* endpoint, or in any other way, it is not mandatory for the Wallet Instance to publish its metadata online if the User does not want this. As a result, the Wallet Instance MUST NOT publish its federation metadata online.

Reviewer: ```suggestion
Given that the Wallet Instance cannot publish its metadata online at the *.well-known/openid-federation* endpoint, 
it MUST obtain a Wallet Instance Attestation issued by its Wallet Provider. The Wallet Instance Attestation MUST contain all the relevant information regarding the security capabilities of the Wallet Instance and its protocol related configuration. It SHOULD contain the Trust Chain related to its issuer (Wallet Provider).
```

> However, the Wallet Instance MUST obtain a Wallet Attestation Instance issued by its Wallet Provider, it SHOULD contain the Trust Chain related to its issuer (Wallet Provider).

Reviewer: ```suggestion
```

- When a Wallet Instance requests the Entity Statements to build the Trust Chain for a specific Relying Party or validates a Trust Mark online, issued for a specific Relying Party, the Trust Anchor or its Intermediate do not know that a particular Wallet Instance is inquiring about a specific Relying Party; instead, they only serve the statements related to that Relying Party as a public resource.
- The Wallet Instance metadata MUST not contain information that may disclose technical information about the hardware used.
- Leaf entity, Intermediate, and Trust Anchor metadata may include the necessary amount of data as part of administrative, technical, and security contact information. It is generally not recommended to use personal contact details in such cases. From a legal perspective, the publication of such information is needed for operational support concerning technical and security matters and the GDPR regulation.

Considerations about Decentralization
-------------------------------------

- There may be more than a single Trust Anchor.

> - In some cases, a trust verifier may trust an Intermediate, especially when the Intermediate may assume the role of the Trust Anchor within a specific perimeter, such as cases where the Leafs are both in the same perimeter like a Member State jurisdiction (eg: an Italian Relying Party with an Italian Wallet Instance may consider the Italian Intermediate as a Trust Anchor for the scopes of their interactions).

Reviewer: ```suggestion
- In some cases, a trust verifier may trust an Intermediate, especially when the Intermediate acts as a Trust Anchor within a specific perimeter, such as cases where the Leafs are both in the same perimeter like a Member State jurisdiction (eg: an Italian Relying Party with an Italian Wallet Instance may consider the Italian Intermediate as a Trust Anchor for the scopes of their interactions).
```

- Trust attestations (Trust Chain) should be included in the JWS issued by Credential Issuers, and the Presentation Requests of RPs should contain the Trust Chain related to them (issuers of the presentation requests).

> - Since the credential presentation must be signed, saving the signed presentation requests and responses, which include the Trust Chain, the Wallet Instance may have the snapshot of the federation configuration (Trust Anchor Entity Configuration in the Trust Chain) and the verifiable reliability of the Relying Party it has interacted with. This information must be stored on the Wallet Device and may be backed up in a remote and secure cloud storage, with the explicit permission of its User.

Reviewer: ```suggestion
- Since the credential presentation must be signed, storing the signed presentation requests and responses, which include the Trust Chain, the Wallet Instance may have the snapshot of the federation configuration (Trust Anchor Entity Configuration in the Trust Chain) and the verifiable reliability of the Relying Party it has interacted with. This information must be stored on the Wallet Device and may be backed up in a remote and secure cloud storage, with the explicit permission of its User.
```
